src/neuromaps_nhp/graph.py

```diff
@@ -0,0 +1,394 @@
+"""Module for managing brain template surface spaces and their transformations."""
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import matplotlib.cm as cm
+import matplotlib.colors as mcolors
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+import yaml
+from matplotlib.lines import Line2D
+
+
+@dataclass
+class Resource(ABC):
+    """Abstract base class for brain template resources."""
+
+    name: str
+    description: str
+    file_path: Path
+
+    @abstractmethod
+    def fetch(self) -> Path:
+        """Fetch the resource."""
+        pass
+
+
+@dataclass
+class Surface_Atlas(Resource):
```

Comment:
```suggestion
class SurfaceAtlas(Resource):
```

Pascal case for classes 😊